### ISSUE TITLE
Tutorial Guide for Verilog Module

### ISSUE THREAD

**Issue by Shivansh2407** [CircuitVerse/CircuitVerse#1944]
**Is your feature request related to a problem? Please describe.**
When I open a New Verilog Module, on clicking the Tutorial Guide, it does not works. Being a new feature, and for everyone to understand about this, We can introduce a separate Tutorial Guide for the Verilog Module that explains each and thing present in the Verilog Module.

**Describe the solution you'd like**
We can introduce a separate Tutorial Guide for the Verilog Module that explains each and every feature about it, as it is being done for the normal circuit.

**Additional context**
We can also provide a sample piece of code during the tutorial for them to get started with the Verilog Coding, this way they can get help easily.

**Are you working on this? (Yes/No)**
No

**Comment by Shivansh2407** [comment]
@satu0king @sal2701 @abstrekt @nitin10s Please let me know your views on this.

**Comment by satu0king** [comment]
yes we can do this. But why stop at this? we can have tutorial guide for edit layout mode as well. Also maybe tutorial guides for certain features like the timing diagram feature.

**Comment by Shivansh2407** [comment]
Oh so Here I was talking to implement a simulator guide as we already have one, for the verilog module, to know and understand better about the Verilog Feature in the Simulator. Because on clicking the Tutorial guide button in drop-down, in the normal circuit starts the guide but the same does not works in the Verilog module, so wanted that feature. 
As far as Tutorials are concerned, so yes we need tutorials for Specific features, but the question is, do we need to implement a guide in simulator or Videos Will work? We need to figur this for helping new users getting started with CV Simulator

**Comment by satu0king** [comment]
Videos are there, this is extra @Shivansh2407. But mainly the current tutorial guide is enough, lets see how this goes before we try and confuse our users with multiple guides :P
